app/controllers/decidim/devise/omniauth_registrations_controller.rb

```diff
@@ -77,7 +74,7 @@ def first_login_and_not_authorized?(user)
       end
 
       def action_missing(action_name)
-        return send(:create) if devise_mapping.omniauthable? && User.omniauth_providers.include?(action_name.to_sym)
+        return send(:create) if devise_mapping.omniauthable? && current_organization.enabled_omniauth_providers.keys.include?(action_name.to_sym)
```

Comment:
What this change stands for @victorol1 ?
I couldn't find the PR or upgrade note anywhere..

Reply:
Since this file is an override it's the newest code from version 0.21-stable. Have in mind that any file modified under the app folder when upgrading is usually an override.